src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,38 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 120;
+    private static final int MIN_LETTERS = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throwRuntimeException("User can't be null");
+        }
+        if (user.getAge() == null) {
+            throwRuntimeException("Age can't be null");
+        } else if (user.getAge() < MIN_AGE || user.getAge() > MAX_AGE) {
+            throwRuntimeException("Age have be wright");
```

Comment:
better have more informative messages - like in your tests: "Age has to be between 18 and 120"